zombies.js

```diff
@@ -79,6 +90,43 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+// class Player {
+//   constructor(name, health, strength, speed) {
+//     this.name = name;
+//     this.health = health;
+//     this.strength = strength;
+//     this.speed = speed;
+//     this.isAlive = true;
+//     this.equipped = false;
+//     this._pack = [];
+//     this._maxHealth = health;
+//   }
+
+//   getPack() {
+//     return this._pack;
+//   }
+
+//   getMaxHealth() {
+//     return this._maxHealth;
+//   }
+
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+  this._pack = [];
+  this._maxHealth = health;
```

Comment:
`_pack` and `_maxHealth` should be placed on a const so that it is truly private.